x/treasury/types/params.go

```diff
@@ -250,5 +254,23 @@ func validateBurnTaxSplit(i interface{}) error {
 		return fmt.Errorf("burn tax split can not greater than 1")
 	}
 
+	return nil
+
+}
+
+func validateMinInitialDepositRatio(i interface{}) error {
+	v, ok := i.(sdk.Dec)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+	
+	if v.GT(sdk.OneDec()) {
+		return fmt.Errorf("min initial deposit ratio is > 1.0")
+	}
+
+	if sdk.ZeroDec().GT(v) {
+		return fmt.Errorf("min inital deposit ratio is < 0.0")
```

Comment:
Maybe the linter wants to be a gramma teacher? :D

Reply:
Error strings should not be capitalized? What is that supposed to mean?

Reply:
Linter is happy now